README.md

```diff
@@ -70,6 +71,19 @@ end
 
 More details on https://hexdocs.pm/tower/Tower.html#module-manual-reporting.
 
+### Configuration
+
+`TowerSentry` supports the following configuration options:
+
+- `:dsn` (`t:String.t/0`) - The DSN for your Sentry project. Setting this option is mandatory. Learn more about DSNs in the [official Sentry documentation](https://docs.sentry.io/concepts/key-terms/dsn-explainer/).
+- `:environment_name` (`t:String.t/0` or `t:atom/0`) - The current environment name. Setting this option is mandatory. Learn more about environments in the [official Sentry documentation](https://docs.sentry.io/concepts/key-terms/environments/).
```

Comment:
I hadn't initially realized that `:environment_name` was mandatory (it's retrieved through `fetch_env!/2`). This is out of scope for this PR but I think sentry's default of using `"production"` makes sense. Maybe TowerSentry could do the same?

Reply:
`fetch_evn!` calls won't fail if not set because we have defaults in https://github.com/mimiquate/tower_sentry/blob/0982d8473d967599ca531b0aae3757e8dd2de714/mix.exs#L33.

Reply:
So in the end `dsn` is mandatory, but `environment_name` will default to `"production"`.

Reply:
Oh I wasn't aware you could set defaults like that way. That's good to know! I'll update the docs to reflect this.